external-plugins/referee/server/server.go

```diff
@@ -247,10 +247,14 @@ func (s *Server) handlePullRequestComment(ic github.IssueCommentEvent) error {
 		}
 	}
 
+	// we need to notify the PR author that his PR is being held, otherwise we might notify a bot user
```

Comment:
nit: I'm not sure we need the comment here? The commit covers it.

Reply:
Fair point!